include/odp/api/spec/classification.h

```diff
@@ -581,6 +583,14 @@ typedef enum {
 	/** Destination SCTP port (val_sz = 2) */
 	ODP_PMR_SCTP_DPORT,
 
+	/** GTPv1 tunnel endpoint identifier (val_sz = 4)
+	 *
+	 * Matches if and only if IP protocol is UDP, UDP destination port
+	 * is 2152 and the UDP payload interpreted as GTP header has GTP
+	 * version 1 and TEID as specified.
+	 */
+	ODP_PMR_GTP_TEID,
```

Comment:
Since this is only for GTP v1, should we call it e.g. ODP_PMR_GTP1_TEID or ODP_PMR_GTPV1_TEID. This would make it more explicit when using the enum that this is only GTP V1 and would be similar to ODP_PMR_DIP_ADDR vs ODP_PMR_DIP6_ADDR.

Reply:
@psavol  @JannePeltonen should I change it to  ODP_PMR_GTPV1_TEID?

Reply:
I think it would be good. Can you also add a validation test that checks that GTPv2 packes do not match?